openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -101,4 +99,9 @@ jQuery(function () {
     initAnalytics($);
     init($);
     initShowPasswords($);
+    // conditionally load functionality based on what's in the page
+    if (document.getElementById('editions')) {
+        import(/* webpackChunkName: "editions-table" */ './editions-table')
+            .then((module) => module.initEditionsTable() );
```

Comment:
Can drop the `()`
```suggestion
            .then(module => module.initEditionsTable());
```

Reply:
Done

> > `import()` calls use promises internally. If you use `import()` with older browsers, remember to shim `Promise` using a polyfill such as `es6-promise` or `promise-polyfill`.
> 
> Is this an issue for us? IE11 doesn't support promises.

We support IE11 so yes this matters and yes this does break IE11. Excellent catch! `promise-polyfill` is the most lightweight of these solutions so I'll go with that.